crew_testing/config.py

```
@@ -0,0 +1,337 @@
+        return sum(
+            config.get("cost_per_run", 0.0)
+            for config in cls.AGENT_LLM_CONFIG.values()
+        )
+
+    @classmethod
+    def print_llm_config(cls) -> None:
+        """Print current LLM configuration for all agents"""
+        print("\n🤖 MT Testing Crew - LLM Configuration")
+        print("=" * 70)
+        print(f"\nDefault Provider: {cls.DEFAULT_PROVIDER.value}")
+        print(f"\nProviders Available:")
```

Ruff (0.14.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)